src/components/RepoCard.jsx

```diff
@@ -55,6 +56,10 @@ const useStyles = makeStyles(() => ({
   content: {
      textAlign: "left",
      color: "#606060",
+     maxHeight:"148px"
```

Comment:
try to use % or em/rem instead of hard coding pixels

Reply:
changed all px measurements to rem